CMakeLists.txt

```diff
@@ -142,6 +148,20 @@ else()
   endif()
 endif()
 
+if(UNIX AND NOT APPLE)
```

Comment:
I kept it at `UNIX AND NOT APPLE` since I figured other platforms like FreeBSD can also build with LTO; although I have not tested. If FreeBSD and other UNIXes are not a worry, I can change it to `IF(LINUX)`